validation for description field in pack_metadata file (#1384)
demisto/demisto-sdk: CHANGELOG.md

* Enhanced the **unify** command when giving input of a file and not a directory return a clear error message.
* Added a validation to ensure integrations are not skipped and at least one test playbook is not skipped for each integration or script.
* Added to the Content Tests support for `context_print_dt`, which queries the incident context and prints the result as a json.

> * Added a warning to ensure that pack_metadata description field is no longer than 130 chars.

Reviewer: ```suggestion
* Added a warning when pack_metadata's description field is longer than 130 characters.
```
